assets/stylesheets/layout/_sidebar.scss

```diff
@@ -126,9 +126,7 @@
 		overflow: hidden;
 		display: flex;
 
-		&:focus {
-			outline: none;
-		}
+		-webkit-tap-highlight-color: rgba($gray-dark, .2);
```

Comment:
I've verified it on iOS and Android's stock browser (via Genymotion). I don't think Windows Phone/IE supports the tag quite yet though.